static/src/javascripts/projects/common/views/loyalty/save-for-later--signed-in.html

```diff
@@ -0,0 +1,8 @@
+<div class="meta__save-for-later" data-link-name="meta-save-for-later" data-component="meta-save-for-later">
+    <a class="button button--medium button--tertiary save-for-later__button js-save-for-later-button save-for-later--{{state}}" data-link-name="meta-save-for-later" data-component="meta-save-for-later">
```

Comment:
You already have `data-link-name="meta-save-for-later" data-component="meta-save-for-later"` in the parent div